.codecov.yml

```diff
@@ -0,0 +1 @@
+comment: false
```

Comment:
FYI, this is used to disable comment submitted from codecov, e.g., https://github.com/JuliaImages/ImageContrastAdjustment.jl/pull/30#issuecomment-606639362